jekyll/_cci2/project-walkthrough.md

```diff
@@ -36,7 +36,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/python:3.6.2-stretch-browsers
+      - image: cimg/python:3.6.2-stretch-browsers
```

Comment:
This is extracted from the demo app linked to in the intro text. AFAIU the demo app will need updating to use a next gen image first. Also Browser teating using a different image variant now, as mentioned in the migration guide: https://discuss.circleci.com/t/legacy-convenience-image-deprecation/41034#browser-testing-4

I feel like some of these changes are a little more involved that straight swap to `cimg`.

Reply:
@felicianotech ^^

Reply:
https://github.com/CircleCI-Public/circleci-demo-python-flask/blob/da7d6585eeac60722e04b10f612acb327e9cd72c/.circleci/config.yml#L30